### ISSUE TITLE
dont revect 0x21 and 0x2f by default

### ISSUE THREAD

**Issue by stsp** [stsp/dosemu2#621]
I think we don't need to revect these ints
these days. Even if `fdconfig.sys` doesn't load
`emufs.sys`, the ae00 trick will do the job, but
the config.sys will then run from fatfs:
https://github.com/stsp/dosemu2/commit/0873d7dc34d1f8a35a5738aa6a1c9984cc0e9d23
Is it a problem?

Bart wanted to revect 0x21 for early logging.
This means we can either add an option, or,
for example, revector it depending on a debug
flags.
But not by default.
My own preference seems to be a new option
to turn it on, and also turn on automatically by
appropriate debug flags.

Thoughts?

**Comment by stsp** [comment]
Should probably be done after #587 is solved.

**Comment by stsp** [comment]
int33 is now also revectored...
So if we remove that, emufs.sys will
also be a mouse driver.

**Comment by jschwartzenberg** [comment]
This broke Windows 3.1 (on top of MS-DOS 6.22) on my system.

**Comment by stsp** [comment]
Works perfectly here (both with `/3` and not).
Please open another ticket, describe your config,
attach logs and see how `$_force_int_revect`
affects that.

**Comment by stsp** [comment]
Also if you are not using `emufs.sys`, please
fix your config. :) I won't care about "other DOSes"
any more, other than in the default configuration.
And default configuration includes `emufs.sys`.

Hmm... Or maybe not to kill "other DOSes" too
instantly, and make `$_force_int_revect` auto-reenable
for them? What people do think?

**Comment by stsp** [comment]
```
# force interrupt revectoring
# Use legacy interrupt revector technique to install interrupt hooks,
# You don't need this if you have emufs.sys loaded via config.sys.
# default: off

# $_force_int_revect = (off)
```

But obviously people are not going to read this
and add `emufs.sys` to their config files. :(

**Comment by stsp** [comment]
Should now be "fixed".

**Comment by stuaxo** [comment]
If emufs is loaded and the error happens while in another DOS, is it worth the error message including some info about this ?

**Comment by stsp** [comment]
Not sure what you mean.
If emufs is loaded, there should be no errors in
any other DOS. The problem was that windows
doesn't boot when emufs.sys is NOT loaded.
I returned the old behaviour for "other DOSes"
now, but FreeDOS is still out of this: one needs
emufs.sys under freedos.
